src/pages/aura/faq.md

```
@@ -4,43 +4,15 @@ metaTitle: FAQ | Aura
 description: Frequently asked questions about Aura
 ---
 
-## General
+## How can I access the Aura gateway?
 
-### What use cases inspired Aura?
+The Aura gateway is currently disabled and undergoing a technical review in order to address certain errors and performance issues. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)